controller/search_blackbox_test.go

```diff
@@ -403,6 +401,140 @@ func (s *searchControllerTestSuite) TestSearchFilter() {
 	assert.Equal(s.T(), "specialwordforsearch", r.Attributes[workitem.SystemTitle])
 }
 
+func (s *searchControllerTestSuite) TestSearchByWorkItemTypeGroup() {
+	s.T().Run(http.StatusText(http.StatusOK), func(t *testing.T) {
+		// given
+		fxt := tf.NewTestFixture(t, s.DB, tf.CreateWorkItemEnvironment())
+		svc := testsupport.ServiceAsUser("TestUpdateWI-Service", *fxt.Identities[0])
+		workitemsCtrl := NewWorkitemsController(svc, gormapplication.NewGormDB(s.DB), s.Configuration)
+		// given work items of different types and in different states
+		type testWI struct {
+			Title          string
+			WorkItemTypeID uuid.UUID
+			State          string
+			SpaceID        uuid.UUID
+		}
+		testWIs := []testWI{
+			{"closed feature", workitem.SystemFeature, workitem.SystemStateClosed, fxt.Spaces[0].ID},
+			{"open feature", workitem.SystemFeature, workitem.SystemStateOpen, fxt.Spaces[0].ID},
+			{"closed bug", workitem.SystemBug, workitem.SystemStateClosed, fxt.Spaces[0].ID},
+			{"open bug", workitem.SystemBug, workitem.SystemStateOpen, fxt.Spaces[0].ID},
+			{"open experience", workitem.SystemExperience, workitem.SystemStateOpen, fxt.Spaces[0].ID},
+			{"closed experience", workitem.SystemExperience, workitem.SystemStateClosed, fxt.Spaces[0].ID},
+			{"open task", workitem.SystemTask, workitem.SystemStateOpen, fxt.Spaces[0].ID},
+			{"closed task", workitem.SystemTask, workitem.SystemStateClosed, fxt.Spaces[0].ID},
+			{"open scenario", workitem.SystemScenario, workitem.SystemStateOpen, fxt.Spaces[0].ID},
+			{"closed scenario", workitem.SystemScenario, workitem.SystemStateClosed, fxt.Spaces[0].ID},
+			{"open fundamental", workitem.SystemFundamental, workitem.SystemStateOpen, fxt.Spaces[0].ID},
+			{"closed fundamental", workitem.SystemFundamental, workitem.SystemStateClosed, fxt.Spaces[0].ID},
+		}
+		for _, wi := range testWIs {
+			payload := minimumRequiredCreateWithTypeAndSpace(wi.WorkItemTypeID, wi.SpaceID)
+			payload.Data.Attributes[workitem.SystemTitle] = wi.Title
+			payload.Data.Attributes[workitem.SystemState] = wi.State
+			_, _ = test.CreateWorkitemsCreated(t, svc.Context, svc, workitemsCtrl, wi.SpaceID, &payload)
```

Comment:
you can remove `_, _ = `
Or I will suggest assert actually
```
_, wi := test.CreateWorkitemsCreated(....)
require.NotNil(s.T(), wi.Data)
```

Reply:
This will likely go away the moment we have a test fixture for work item type groups. But for now I like to always make it obvious that I'm ignoring return values. It is safer and screams for improvement this way.